packages/wix-ui-tpa/src/components/Card/docs/index.story.tsx

```diff
@@ -20,6 +20,12 @@ import * as exampleStyles from './examples.st.css';
 import { settingsApi } from '../../../../stories/utils/SettingsApi';
 import { storyComponent } from '../../../../stories/helperComponents/storyComponent';
 import { StoryCategory } from '../../../../stories/storyHierarchy';
+import { settingsPanel } from '../../../../stories/utils/SettingsPanel';
+// @ts-ignore
+import CardWiringExampleRawSource from 'raw-loader!./CardWiringExample';
+// @ts-ignore
+import CardWiringExampleCSSRawSource from 'raw-loader!./CardWiringExample.st.css';
```

Comment:
Maybe like that?
```suggestion
import * as CardWiringExampleRawSource from 'raw-loader!./CardWiringExample';
import * as CardWiringExampleCSSRawSource from 'raw-loader!./CardWiringExample.st.css';
```

Instead of the `// @ts-ignore`